### PR TITLE
Bugfix dataviz popup

### PR DESCRIPTION
**Pull request by tethysgeco**
* Ticket : 
* #2767 --> plot in popup maintains json customization defined in lizmap plugin
* #491 --> plots order in popup respects plot order defined in lizmap plugin
* Tethys S.r.l. www.tethys.srl

### CODE REVIEW

**Comment by Gustry** [comment]
Thanks for the contribution. I will let @mdouchin check this PR

**Comment by rldhont** [comment]
Thanks @tethysgeco 

@nboisteault do cypress tests cover this code ?

**Comment by nboisteault** [comment]
@rldhont No, dataviz part lacks tests.

File: lizmap/www/assets/js/dataviz/dataviz.js [hunk 1]
```diff
@@ -295,7 +296,7 @@ var lizDataviz = function() {
         return pid;
     }
 
-    function buildPlot(id, conf){
+    function buildPlot(id, conf, pid = null){
```
Comment (rldhont) [inline lizmap/www/assets/js/dataviz/dataviz.js:299]: This change uses [Default parameters in function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters). The [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#browser_compatibility) seems OK with Lizmap 3.5 [browser compatibility](https://github.com/3liz/lizmap-web-client/wiki/Versions#web-browsers).

File: lizmap/www/assets/js/dataviz/dataviz.js [hunk 2]
```diff
@@ -295,7 +296,7 @@ var lizDataviz = function() {
         return pid;
     }
 
-    function buildPlot(id, conf){
```
Comment (mdouchin) [inline lizmap/www/assets/js/dataviz/dataviz.js:298]: Please change also the definition of the `buildPlot` function at the end of the file to integrate this new parameter, after
`var obj = {`

**Review by mdouchin — CHANGES_REQUESTED** [review]

File: lizmap/www/assets/js/dataviz/dataviz.js (hunk 2 above)
Comment (rldhont) [reply lizmap/www/assets/js/dataviz/dataviz.js:298]: @mdouchin I have done it.

**Pull request merged**